components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp

```diff
@@ -239,7 +239,14 @@
 
         var stepOrder = 0;
         function nextWizard(){
-            if(!validateInputs()){
+        <%
+        boolean intTemp = true;
+        if(template !=null && template.getName().equals("External Workflow Template")){
```

Comment:
Let's define `"External Workflow Template"` as a constant in the `WFConstant` class and use that constant here.